ckanext/dgu/theme/templates/page.html

```diff
@@ -15,18 +15,16 @@
 #}
 {%- block content %}
     {% block maintag %}<div role="main">{% endblock %}
-      <div id="content">
-        {% block main_content %}
```

Comment:
sorry David, i've been too aggressive in my mail filters.

I completely agree with the fact that we should stick to CKAN structure to ease merges. My mistake. I have modified the file to get back to original order.

Regarding your remark on main_content, i don't get it : i did not override it, it is still in page.html . Did I miss something?